csunplugged/topics/content/en/binary-numbers/unit-plan/lessons/how-binary-digits-work-junior.md

```diff
@@ -0,0 +1,378 @@
+# How Binary Digits Work
```

Comment:
Any existing content before the style guide probably won't abide by it, please raise an issue for any you see.